WebInterface/app/models/setting.rb

```diff
@@ -0,0 +1,27 @@
+class Setting < ApplicationRecord
+  before_save :format_data
+  has_many :dataset, dependent: :destroy
+
+  validate :validate_datetime_fields
+
+  def format_data
+    words = self.models.split(',').map { |word| word.gsub(/[^a-zA-Z ]/, '') }
+    self.models = words.join(' ')
+  end
+
+  private
+
+  def validate_datetime_fields
+    unless DateTime.parse(self.start_date.to_s)
+      errors.add(:start_date, "is not a valid datetime")
+    end
+
+    unless DateTime.parse(self.end_date.to_s)
+      errors.add(:end_date, "is not a valid datetime")
+    end
+
+    if self.start_date.present? && self.end_date.present? && DateTime.parse(self.start_date.to_s) >= DateTime.parse(self.end_date.to_s)
```

Comment:
Er det ikke muligt bare at sige:
```suggestion
    if self.start_date.present? && self.end_date.present? && self.start_date >= self.end_date
```